packages/ckeditor5-clipboard/src/clipboardmarkersutils.ts

```diff
@@ -536,16 +589,25 @@ export default class ClipboardMarkersUtils extends Plugin {
 export type ClipboardMarkerRestrictedAction = 'copy' | 'cut' | 'dragstart';
 
 /**
- * Specifies copy, paste or move marker restrictions in clipboard. Depending on specified mode
- * it will disallow copy, cut or paste of marker in clipboard.
- *
- * 	* `'default'` - the markers will be preserved on cut-paste and drag and drop actions only.
- * 	* `'always'` - the markers will be preserved on all clipboard actions (cut, copy, drag and drop).
- * 	* `'never'` - the markers will be ignored by clipboard.
+ * Specifies behavior of markers during clipboard actions.
  *
  * @internal
  */
-export type ClipboardMarkerRestrictionsPreset = 'default' | 'always' | 'never';
+export type ClipboardMarkerConfiguration = {
+	allowedActions: NonEmptyArray<ClipboardMarkerRestrictedAction> | 'all';
+
+	// If false, do not copy marker when only part of its content is selected.
+	withPartiallySelected?: boolean;
+
+	// If true then every marker that is present in clipboard document fragment element obtain new generated id just before pasting.
```

Comment:
```suggestion
	// If `true` then every marker present in clipboard document fragment element obtains new generated ID just before pasting.
```

Reply:
You sometimes use `ID` and sometimes `id`. Let's keep it consistent as `ID`. You can refactor it in all places.